qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -874,14 +874,14 @@ class TwoQubitBasisDecomposer:
     basis gate.
 
     Args:
-        gate (Gate): Two-qubit gate to be used in the KAK decomposition.
-        basis_fidelity (float): Fidelity to be assumed for applications of KAK Gate. Default 1.0.
-        euler_basis (str): Basis string to be provided to OneQubitEulerDecomposer for 1Q synthesis.
-            Valid options are ['ZYZ', 'ZXZ', 'XYX', 'U', 'U3', 'U1X', 'PSX', 'ZSX', 'RR'].
-        pulse_optimize (None or bool): If True, try to do decomposition which minimizes
+        gate: Two-qubit gate to be used in the KAK decomposition.
+        basis_fidelity: Fidelity to be assumed for applications of KAK Gate. Default 1.0.
+        euler_basis: Basis string to be provided to OneQubitEulerDecomposer for 1Q synthesis.
+            Valid options are ``['ZYZ', 'ZXZ', 'XYX', 'U', 'U3', 'U1X', 'PSX', 'ZSX', 'RR']``.
+        pulse_optimize: If ``True``, try to do decomposition which minimizes
             local unitaries in between entangling gates. This will raise an exception if an
-            optimal decomposition is not implemented. Currently, only [{CX, SX, RZ}] is known.
-            If False, don't attempt optimization. If None, attempt optimization but don't raise
+            optimal decomposition is not implemented. Currently, only ``[{CX, SX, RZ}]`` is known.
```

Comment:
I'm not exactly clear what the inline code here is trying to say.

Reply:
I think I went a bit into automatic mode, we can probably leave the formatting out for this one.